ProductsManegements/Inventory.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System.Diagnostics;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+namespace SimpleInventoryManagementSystem.ProductsManegements
+{
+    internal class Inventory
+    {
+        internal static List<Product> products =new();
+
+        public void AddProduct()
+        {
+            string name = " ";
```

Comment:
Redundant code. No need for this initial assignment  since the name value will be replaced in the next code line 
`string name = Console.ReadLine() ?? string.Empty;`